### DISCUSSION TITLE
The most up-to-date MACCS166 fingerprint (after 2010, next to 2021)

### DISCUSSION THREAD

**Discussion by WelingtonSilvaDev** [rdkit/rdkit#3946] [Development]
I was searching for the **MACCS166** fingerprint [BUTINA] with the features most updated, but I couldn't find anything. The last and best definition of each descriptor is on the link from Open Babel: http://www.scbdd.com/pybel_desc/fps-maccs/

Someone knows one good source for find MACCS166 updated descriptors definition?

**Comment by greglandrum** [comment]
Here's the RDKit version of that: https://github.com/rdkit/rdkit/blob/bc4d5478478542045a0f89e28a5da8b6d1f0e5d0/rdkit/Chem/MACCSkeys.py
And that's the only publicly available form that I know.
It used to be something that you could find in the ISIS/Sketch documentation, but I haven't seen any other source of those.

**Reply by WelingtonSilvaDev** [reply]
Great, thank you! :D
I also hadn't found updated in any original documentation source. There're only old records, dated 2010 down.

**Reply by greglandrum** [reply]
The 166 MACCS keys are quite old, so it's not surprising you can't find a reference.
The only thing public I'm aware of that's even close is this: https://pubs.acs.org/doi/full/10.1021/ci010132r
and that just mentions the 166 keys, but doesn't define them.
